python_modules/libraries/dagster-sdf/dagster_sdf/sdf_information_schema.py

```diff
@@ -289,10 +290,10 @@ def _extract_code_ref(
     def stream_asset_observations(
         self,
         dagster_sdf_translator: DagsterSdfTranslator,
-        context: Optional[OpExecutionContext] = None,
+        context: Optional[Union[OpExecutionContext, AssetExecutionContext]] = None,
     ) -> Iterator[SdfDagsterEventType]:
         selected_output_names: AbstractSet[str] = (
-            context.selected_output_names if context else set()
+            context.op_execution_context.selected_output_names if context else set()
```

Comment:
`AssetExecutionContext` raises a deprecation warning if `context.selected_output_named` is called, but allows you to get the underlying `op_execution_context`. `OpExecutionContext` also implements `op_execution_context`